kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ResourcesUtil.java

```diff
@@ -126,6 +138,15 @@ public static String uid(@NonNull HasMetadata resource) {
         return resource.getMetadata().getUid();
     }
 
+    public static String coordinatesString(LocalRef<?> ref, HasMetadata primary) {
```

Comment:
* "coordinatesString" is such a mouthful to say for such a simple thing. 
* What `primary` is really doing here it providing the namespace, so maybe there's a better name. `inNamespaceOf`?

Reply:
done, I've shuffled it into the `LocalRef` hierarchy so now you can say `NamespacedLocalRef<T> LocalRef<T>#inNamespaceOf(HasMetadata resource)` and have made this the toString.

hmm, `NamespacedLocalRef` doesn't really fit, `LocalRef` is docced as being in the same namespace. Maybe it should be just `NamespacedRef` and it should have equals/hash that take namespace into account too.